index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Hello Microverse</title>
+</head>
+
+<body>
+	<header>
+		<h1 class="page_head">Hello Microverse!</h1>
+	</header>
```

Comment:
[OPTIONAL[
This is an example inline comment which you have used. :slightly_smiling_face: 

But you can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. wink

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.